### ISSUE TITLE
Azure Data Studio no longer working on MacOS 10.14.6

### ISSUE THREAD

**Issue by chipfelton** [microsoft/azuredatastudio#6818]
Issue Type: <b>Bug</b>

I have been using ADS on MacOS 10.14 w/o issues until today, when upon opening the app numerous components failed to start. 

Azure Data Studio version: Azure Data Studio 1.10.0 (5fdb967ffcea41df0335ec17cfa396a694bb174e, 2019-08-14T18:12:56.165Z)
OS version: Darwin x64 18.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-3740QM CPU @ 2.70GHz (8 x 2700)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.37GB free)|
|Process Argv|-psn_0_766139|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
admin-pack|Mic|0.0.2
agent|Mic|0.41.0
cms|Mic|0.4.0
dacpac|Mic|0.5.0
powershell|ms-|2019.5.0
profiler|Mic|0.9.0
server-report|Mic|0.1.5
whoisactive|Mic|0.1.1

</details>

<img width="578" alt="Welcome_—_Untitled__Workspace_" src="https://user-images.githubusercontent.com/2133943/63309541-42ab5900-c34b-11e9-8815-967e636113b0.png">
<img width="1436" alt="Welcome_—_Untitled__Workspace_" src="https://user-images.githubusercontent.com/2133943/63309571-59ea4680-c34b-11e9-8adf-45c1bbc24236.png">

**Comment by kburtram** [comment]
@chipfelton have you tried reinstalling Azure Data Studio?  I'm not aware of any changes that would prevent ADS from working on MacOS 10.14.

**Comment by chipfelton** [comment]
Hi Karl,

I did try that but encountered the same problems.  However, I then uninstalled and instead installed the “Insiders” version (1.10.0-insider) and so far that version is working fine.  Many thanks for checking in on me.

Cheers,
Chip

**Comment by chipfelton** [comment]
Hi again,

I believe I’ve now figured out what caused the original problem, ‘cause it happened again with the “Insider” version but this time I managed to track down the source.  I had been saving my work in a set of local folders synced with DropBox.  I recently had to reinstall DropBox and in the process changed the name of my root DropBox folder from “DropBox (Personal)” to “DropBox”.  The extensions that failed in ADS were all looking for folders under the old “DropBox (Personal)” name and changing those in “Settings” to point to the new name “DropBox” solved the problem.  I imagine this would work just the same if I had stayed with the current non-Insider version of ADS.

Cheers,
Chip
